src/appleseed/renderer/kernel/rendering/renderercomponents.cpp

```diff
@@ -443,6 +467,10 @@ bool RendererComponents::create_tile_renderer_factory()
         m_tile_renderer_factory.reset(new DebugTileRendererFactory());
         return true;
     }
+    else if (name.empty())
```

Comment:
Keep the empty case first like it was done before, and like it's done in other similar functions.